src/interface/IDimoCredit.sol

```diff
@@ -7,6 +7,7 @@ interface IDimoCredit {
     function burn(address from, uint256 amount) external;
     function transferFrom(address from, address to, uint256 value) external returns (bool);
     function mintAmountDc(address to, uint256 dimoCredits, bytes calldata data) external returns (uint256 amountIn);
-    function mint(address to, uint256 amountIn, bytes calldata data) external returns (uint256 dimoCredits);
+    function mintInDimo(address to, uint256 amountDimo, bytes calldata data) external returns (uint256 dimoCredits);
+    function mint(address to, uint256 amountDimoCredits, bytes calldata data) external;
```

Comment:
This seems like a breaking change for the `mint` function, doesn't it?

Reply:
I guess the question is whether anyone is calling this. Who is there? Dev console? Can you check what they're using?

Reply:
Oh yeah I am aware of that. Pretty much dev console so far. I would have them update it before deploying.
To clarify, I think keeping `mint` for DCX amout makes more sense and make the distinction for DIMO tokens.